server/config-example.js

```diff
@@ -34,4 +34,4 @@ config.couchDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchDbSer
 config.couchAuthDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchCredentials() + config.couchDbServer + ':' + config.couchDbPort;
 // config.searchURL = 'http://localhost:9200'; ELASTIC SEARCH URL (OPTIONAL)
 config.serverInfo = 'Development Ember CLI server';
-module.exports = config;
+module.exports = config;
```

Comment:
After stopping and thinking it through, it seems like ` _executeContainsSearch` is the only method that makes elastic search requests? I know that's a pretty basic question, but this is  pretty abstracted out for my simple brain.